app/services/metrics/code_climate_summary_retriever.rb

```diff
@@ -0,0 +1,13 @@
+module Metrics
+  class CodeClimateSummaryRetriever < BaseService
+    attr_reader :project_id
+
+    def initialize(project_id)
+      @project_id = project_id
+    end
+
+    def call
+      CodeClimateProjectMetric.joins(:project).where(projects: { id: project_id }).first
```

Comment:
Could we use `find_by` right?
`CodeClimateProjectMetric.joins(:project).find_by(projects: { id: project_id })`

Reply:
Indeed.
Fixed it. Thanks!